core/versionutil.go

```diff
@@ -54,7 +55,7 @@ var ModLoaders = map[string]ModLoaderComponent{
 	"neoforge": {
 		Name:              "neoforge",
 		FriendlyName:      "NeoForge",
-		VersionListGetter: FetchMavenVersionPrefixedListStrip("https://maven.neoforged.net/releases/net/neoforged/forge/maven-metadata.xml", "NeoForge"),
+		VersionListGetter: FetchMavenWithNeoForgeStyleVersions("https://maven.neoforged.net/releases/net/neoforged/neoforge/maven-metadata.xml", "NeoForge"),
```

Comment:
This new metadata.xml has neoforge versions for minecraft 1.20.2+, but the old xml still has the neoforge versions for 1.20.1, can we implement some kind of check to route to the correct xml

Reply:
joining to a single list might be easier and more in line with how the code is structured